boot/freeldr/freeldr/lib/mm/meminit.c

```diff
@@ -262,6 +262,7 @@ MmCheckFreeldrImageFile(VOID)
         (FileHeader->NumberOfSymbols != 0) ||       //    ""      ""
         (FileHeader->SizeOfOptionalHeader != sizeof(IMAGE_OPTIONAL_HEADER)))
     {
+    #ifndef UEFIBOOT
```

Comment:
I would recommend to never indent #ifdefs / #ifndefs / #endifs. Place them at column 0.